lib/rubocop/formatter/disabled_config_formatter.rb

```diff
@@ -23,6 +23,12 @@ class << self
         attr_accessor :config_to_allow_offenses
       end
 
+      def file_started(file, file_info)
```

Comment:
I don't like passing in the args here -- I wish that #started took the cli_options instead of #file_started. But I didn't want to change the API.